iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -10,30 +10,31 @@ import Foundation
 
 class MessageCell: UICollectionViewCell {
 
-    private let messageContainerView = UIView()
-    private let messageTextView = TextView()
+    private lazy var collectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        return UICollectionView(frame: .zero, collectionViewLayout: layout)
+    }()
+    private let cellRegistration = UICollectionView.CellRegistration<MessageSubcell, Messageable>
+    { (cell, indexPath, item) in
+        cell.setText(with: item)
+    }
```

Comment:
I opted for a collectionview to handle the stack. This should make it easier to adjust the number of shown replies and change the layout.